docs/scripts/ExePacking.iss

```diff
@@ -4,7 +4,7 @@
 ; Download Inno Setup: https://jrsoftware.org/isdl.php
 
 #define MyAppName "ArkPets"
-#define MyAppVersion "3.2.0"
+#define MyAppVersion "3.2.1"
```

Comment:
ISS Version Const should not bump without Gradle Version Const bumping and Java Version Const bumping.

Reply:
Oh, s***, I forgot it